### PR TITLE
Schedule

### PR DESCRIPTION
**Pull request by Rogal255**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: schedule-bonus
🏅 Granted 10 XP for @Rogal255

**Comment by ziobron** [comment]
⏱ 3 XP extra for @Rogal255 for on-time delivery

**Comment by ziobron** [comment]
✅ CI Passed - Task: schedule
🏅 Granted 12 XP for @Rogal255

**Comment by ziobron** [comment]
⏱ 3 XP extra for @Rogal255 for on-time delivery

File: homework/schedule/schedule.hpp [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <chrono>
+#include <thread>
+
+template <typename F, typename... Args>
+void schedule(F&& func, const std::chrono::seconds time, const Args&... args) {
+    std::this_thread::sleep_for(time);
+    func(args...);
```
[the remark this answers is not in the document]
Comment (nauka-programowania-MA) [reply homework/schedule/schedule.hpp:9]: Or invoke them into the function using perfect forwarding.

std::invoke(f, std::forward<Args>(args)...);

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**